test/e2e/framework/pod/create.go

```diff
@@ -166,8 +166,12 @@ func MakeSecPod(podConfig *Config) (*v1.Pod, error) {
 	if podConfig.NS == "" {
 		return nil, fmt.Errorf("Cannot create pod with empty namespace")
 	}
-	if len(podConfig.Command) == 0 && !NodeOSDistroIs("windows") {
-		podConfig.Command = "trap exit TERM; while true; do sleep 1; done"
+	if len(podConfig.Command) == 0 {
+		if !NodeOSDistroIs("windows") {
+			podConfig.Command = "trap exit TERM; while true; do sleep 1; done"
+		} else {
+			podConfig.Command = "while true; do sleep 1; done"
```

Comment:
what shell is running on the Windows nodes?
given `while true; do sleep 1; done` i would assume its a Unix-like shell, but why `trap exit TERM; ` can't be used?

Reply:
I am not very sure, from previous testing, it does not work well for Windows.

Reply:
I will try again, and update the details

Reply:
actually I tried "trap exist TERM" on Windows, it is working fine now. Updated the code